auth/client-api.md

```diff
@@ -0,0 +1,22 @@
+# Client API
+
+```typescript
+abstract class Client {
+  // ---------- Methods ----------------------------------------------- //
+
+  // initializes the client with persisted storage and a network connection
+  public abstract init(): Promise<void>;
+
+  // authenticate wallet users with blockchain account for provided chainId
+  public abstract authenticate(params: {
+    chainId: string;
+    nonce: string;
+    aud?: string;
+    nbf?: string;
+    exp?: string;
+    statement?: string;
+    requestId?: string;
+    resources?: string[];
+  }): Promise<boolean>;
```

Comment:
could params be `requestId` aka `requester.publicKey` or topic  and `SignedCACAO` ?
I assume this method is for wallet.